source/tutorials/getting-started-rails6.txt

```diff
@@ -14,11 +12,11 @@ Getting Started (Rails 6)
 
 .. note::
 
-  This tutorial is for Ruby on Rails 6. If this is not the version you're using choose
-  the appropriate tutorial for your Rails version from the navigation menu.
+   This tutorial is for Ruby on Rails 6. If this is not the version you're using choose
+   the appropriate tutorial for your Rails version from the navigation menu.
 
 New Application
-===============
+---------------
 
 This section shows how to create a new Ruby on Rails application using Mongoid
 for data access. The application will be similar to the blog application
```

Comment:
Not part of this PR, but the period in the repo link jumps to the next line. Can fix by moving it to inside the link text.